README.md

```diff
@@ -1 +1,10 @@
-# hw01
\ No newline at end of file
+### Self-Introduction  
+
+![](images/profile_pic.png)  
+
+Hi! My name is **Izzy Allum**, and I am a first year at the University of Chicago. I'm from Washington, DC, so that's where I've been spending quaratine. My hobbies include:  
+
+* fencing (I'm on the club team at UChicago)
+* baking and cooking
+* listening to [Spotify](https://open.spotify.com/) and watching [Netflix](https://www.netflix.com/browse)
```

Comment:
This is a really good way for people to track the information you shared with a link attached! Nice job!